100-sqlvm-setupkeyvault/README.md

```diff
@@ -0,0 +1,11 @@
+# This template setup Azure Key Vault on any existing Azure Virtual machine with SQL Server Standard or Enterprise edition.
```

Comment:
Please rename the template to 101-sql-vm-keyvault-setup for better readability. Another options is to rename to 101-vm-sql-keyvault-setup so it fits in with the other 101-vm templates in the repo.